packages/components/src/public/QueryModel/SaveViewModal.tsx

```diff
@@ -133,13 +134,13 @@ export const SaveViewModal: FC<Props> = memo(props => {
         try {
             const name = isDefaultView ? '' : viewName.trim();
             const isCurrentView = name?.toLowerCase() === currentView?.name?.toLowerCase();
-            await onConfirmSave(name, canInherit, isCurrentView, isDefaultView);
+            await onConfirmSave(name, canInherit, isCurrentView, isDefaultView || isShared);
```

Comment:
just to make sure I remember the desired behavior here, `isDefaultView || isShared` means that any save to the default view will be a "shared default view", i.e. we decided not to allow for private default views in the app right? And the added part is that we restrict who can save default views, right?

Reply:
Yes, no private default views. The addition of `isShared` is just to bring through the choice for sharing from the checkbox in the UI. We don't offer that checkbox for default views, so they are all shared.